src/wazuh_modules/wm_ciscat.c

```diff
@@ -131,6 +180,24 @@ void* wm_ciscat_main(wm_ciscat *ciscat) {
 
             for (eval = ciscat->evals; eval; eval = eval->next) {
                 if (!eval->flags.error) {
+
+                    switch (wm_relative_path(eval->path)) {
+                        case 0:
+                            snprintf(bench_fullpath, OS_MAXSTR - 1, "%s", eval->path);
+                            break;
+                        case 1:
+                        #ifdef WIN32
+                            snprintf(bench_fullpath, OS_MAXSTR - 1, "%s\\%s", cis_path, eval->path);
+                        #else
+                            snprintf(bench_fullpath, OS_MAXSTR - 1, "%s/%s", cis_path, eval->path);
+                        #endif
+                            break;
+                        default:
+                            mterror(WM_CISCAT_LOGTAG, "Couldn't find benchmark path. Skipping...");
+                    }
+
+                    os_strdup(bench_fullpath, eval->path);
```

Comment:
May duplicate trash data (very long string) if the path is empty: `wm_relative_path` would return `-1` and `bench_fullpath` won't be initialized.